components/brave_wallet_ui/common/constants/local-storage-keys.ts

```diff
@@ -9,7 +9,6 @@ export const LOCAL_STORAGE_KEYS = {
   HIDE_PORTFOLIO_NFTS_TAB: 'BRAVE_WALLET_HIDE_PORTFOLIO_NFTS_TAB',
   PORTFOLIO_ASSET_FILTER_OPTION: 'PORTFOLIO_ASSET_FILTER_OPTION',
   PORTFOLIO_TIME_LINE_OPTION: 'PORTFOLIO_TIME_LINE_OPTION',
-  IS_IPFS_BANNER_HIDDEN: 'BRAVE_WALLET_IS_IPFS_BANNER_HIDDEN',
```

Comment:
maybe move to the deprecated list so we can delete the data from the user's local storage?